src/tests/ftest/util/server_utils.py

```diff
@@ -203,8 +203,6 @@ def prepare(self, storage=True):
         if storage:
             # Prepare server storage
             if self.manager.job.using_nvme or self.manager.job.using_dcpm:
-                self.log.info("Preparing storage in <format> mode")
```

Comment:
Is the storage prepare being done elsewhere? Or do we not need to do it at all in ftest?

Reply:
we don't need to do it as it's always performed during daos_server start-up (which first resets device bindings before rebinding)